Stopped at <exception thrown>
_exception :: e = SomeException (GHC.Exception.:DException _
                                                           (GHC.Show.:DShow ...) ....)
                                _
Stopped at <exception thrown>
_exception :: e = SomeException (GHC.Exception.:DException _
                                                           (GHC.Show.:DShow ...) ....)
                                _
*** Exception: user error (error)
Stopped at <exception thrown>
_exception :: e = SomeException (GHC.Exception.:DException _
                                                           (GHC.Show.:DShow ...) ....)
                                (GHC.IOBase.IOError Nothing GHC.IOBase.UserError ....)
*** Exception: user error (error)
Stopped at <exception thrown>
_exception :: e = SomeException (GHC.Exception.:DException _
                                                           (GHC.Show.:DShow ...) ....)
                                (GHC.IOBase.IOError Nothing GHC.IOBase.UserError ....)
Stopped at <exception thrown>
_exception :: e = SomeException (GHC.Exception.:DException _
                                                           (GHC.Show.:DShow ...) ....)
                                (GHC.IOBase.IOError Nothing GHC.IOBase.UserError ....)
